sharedbuffers/mapped_struct.py

```diff
@@ -638,6 +718,8 @@ def __init__(self, value = None, typ = None, TYPE_CODES = TYPE_CODES):
     unicode : mapped_unicode,
     bytes : mapped_bytes,
     object : mapped_object,
+    datetime : mapped_datetime,
+    Decimal : mapped_decimal
```

Comment:
Missing `,`

Also... you should try to import cdecimal and support it in addition to decimal. Same format, same everything, but it's a different class so it has to be included in the type mappings.